lredis/protocol.lua

```diff
@@ -3,7 +3,9 @@
 
 -- Encode a redis bulk string
 local function encode_bulk_string(str)
-	assert(type(str) == "string")
+	if type(str) ~= "string" then
+      str = tostring(str)
```

Comment:
my sneaky snek has failed : )

(actually, i just goofed by including this here)